spec/features/idv/account_creation_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+feature 'account creation after LOA3 request', idv_job: true do
+  include SamlAuthHelper
+  include IdvHelper
+
+  context 'successful IdV with same phone as 2FA' do
```

Comment:
The idea is that there will be several `contexts` in this spec. For example, another one could be choosing USPS verification.